src/integration-test/java/developerTC/TestAssetRegistration_08.java

```diff
@@ -124,10 +126,9 @@ public void testHashForRemoteAsset() throws IOException {
         Asset memoryAsset = MemoryAsset.create(content.getBytes());
 
         DataAsset dataAsset = remoteAgent.uploadAsset(memoryAsset);
-        DataAsset dataAssetWithHash = dataAsset.includeContentHash();
+        dataAsset.includeContentHash();
 
-        Assertions.assertEquals(Hex.toString(Hash.sha3_256(dataAsset.getContent())), Hex.toString(Hash.sha3_256(content)));
-//        assertEquals(Hex.toString(Hash.sha3_256(dataAssetWithHash.getMetadata().get(CONTENT_HASH).toString())), Hex.toString(Hash.sha3_256(content)));
+        Assertions.assertEquals(Hash.checkSum(dataAsset.getContentStream()), dataAsset.getMetadata().get(Constant.CONTENT_HASH));
```

Comment:
How do we know this worked? Use the exact string hash value to check for correctness.